coccinelle/run-coccinelle.sh

```diff
@@ -10,7 +11,7 @@ case "$1" in
                 ;;
 esac
 
-for SCRIPT in ${@-coccinelle/*.cocci} ; do
+for SCRIPT in $top/coccinelle/*.cocci ; do
```

Comment:
This removes the ability to run a single script or some subset of them. I use this :(

Reply:
hmm, will fix